ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -253,3 +253,33 @@ ${local.skip_on_weekends}
     action_group = var.action_group_ids
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "bulk_results_upload" {
+  name                = "${var.env}-bulk_results_upload"
+  description         = "${local.env_title} alert when bulk uploads fail"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+
+  action {
+    action_group = var.action_group_ids
+  }
+
+  data_source_id = var.app_insights_id
+  enabled        = contains(var.disabled_alerts, "bulk_results_upload") ? false : true
+
+  query = <<-QUERY
+requests
+| where true
+| where timestamp >= ago(5m)
+    and name == "/upload/results"
+    and toint(resultCode) >= 500
+  QUERY
+
+  severity    = 1
+  frequency   = 5
+  time_window = 5
+  trigger {
+    operator  = "GreaterThan"
+    threshold = 0
```

Comment:
Outside of the report stream outage, do we have any instances where this alert would have fired? How often?

I see you added the time window and the timestamp in the query :smile: :watch:

Reply:
To be more specific - I ran a query for the last 7 days and didn't see any occurrences where this error would have fired. (In addition to running it during the RS outage)

Reply:
Normally I would just assume that I butchered the query, but I did find the one instance during the outage 🙃

Reply:
`time_window` is part of the scheduled alert config. `time_window` informs the alert how far back it should go to gather data. Without the timestamp in your query, your query runs against all the data available in that window. Adding the timestamp is an additional filter on the data.